### DISCUSSION TITLE
#night mode: how to set it in night mode?

### DISCUSSION THREAD

**Discussion by hojojiojo** [sumatrapdfreader/sumatrapdf#3264] [Q&A]
just as the title shows, how to set it in night mode?

**Comment by GitHubRulesOK** [comment]
Night Light Mode 
![image](https://user-images.githubusercontent.com/12754913/217555535-c13c80f8-9bf3-40c1-9467-2585c775ccad.png)
is a Windows feature for reducing eye strain by REDUCING the more harmful retina burning BLUE light from the screen (making everything pink and low contrast) so it is found in windows settings and Thus SumatraPDF relies on Windows to provide "Night Mode"

To emulate that you could alter settings to
MainWindowBackground = #80fff8f8
and for some contents also set
	BackgroundColor = #fff8f8

However it will have no effect on image formats so better to use the Windows hardware mode settings

![image](https://user-images.githubusercontent.com/12754913/217560736-462b1be3-7f27-4abd-b8f1-2e587c2fc241.png)

**Reply by hojojiojo** [reply]
> Night Light Mode ![image](https://user-images.githubusercontent.com/12754913/217555535-c13c80f8-9bf3-40c1-9467-2585c775ccad.png) is a Windows feature for reducing eye strain by REDUCING the more harmful retina burning BLUE light from the screen (making everything pink and low contrast) so it is found in windows settings and Thus SumatraPDF relies on Windows to provide "Night Mode"
> 
> To emulate that you could alter settings to MainWindowBackground = #80fff8f8 and for some contents also set BackgroundColor = #fff8f8
> 
> However it will have no effect on image formats so better to use the Windows hardware mode settings
> 
> ![image](https://user-images.githubusercontent.com/12754913/217560736-462b1be3-7f27-4abd-b8f1-2e587c2fc241.png)

i have tried this method, it doesn't work. Is there any other way?

**Comment by dungsaga** [comment]
You can use a dark color theme for night mode:
![image](https://user-images.githubusercontent.com/5884951/226249977-78481eef-cc55-4e37-8f66-2a95c44d0da7.png)


And press "i" to invert it to light mode:
![image](https://user-images.githubusercontent.com/5884951/226249922-d4f3fbcf-f8d2-4ddf-9f9a-d4d6a8311551.png)
